src/sinks/pulsar.rs

```diff
@@ -45,7 +45,8 @@ enum BuildError {
 #[configurable_component(sink("pulsar"))]
 #[derive(Clone, Debug)]
 pub struct PulsarSinkConfig {
-    /// The endpoint to which the Pulsar client should connect to.
+    /// The endpoint to which the Pulsar client should connect to. The endpoint should specify the
+    /// pulsar protocol and port.
```

Comment:
nit: we generally seem to be line breaking these. See

https://github.com/vectordotdev/vector/blob/2bfd63922e7d82f7616138b51251f27a3052c9b4/src/sources/splunk_hec/mod.rs#L62-L64